.github/workflows/CI.yml

```diff
@@ -0,0 +1,65 @@
+name: full_tests
+
+on:
+  push:
+    branches:
+      - main
+      - master
```

Comment:
I have no idea how @richardjgowers managed to create a repo with `master` by default 🤣

Reply:
Haha this was off the github setup, I thought they'd transitioned to main but apparently not.

Reply:
Do you want me to change the branch name from master to main?

Reply:
Yeah no reason not to.